CMakeLists.txt

```diff
@@ -745,8 +745,9 @@ elseif(ANDROID)
   )
 
   if(LOVR_BUILD_EXE)
-    set(ANDROID_JAR "${ANDROID_SDK}/platforms/${ANDROID_PLATFORM}/android.jar")
-    set(ANDROID_TOOLS "${ANDROID_SDK}/build-tools/${ANDROID_BUILD_TOOLS_VERSION}")
+    find_package(Java)
```

Comment:
Note: This already exists in the "Setup" block near the top of the file.

Reply:
Thanks. This was a merge error.